package.json

```diff
@@ -134,5 +134,9 @@
       "eqeqeq": "error",
       "prefer-arrow-callback": "error"
     }
+  },
+  "resolutions": {
```

Comment:
TODO: remove patches when https://github.com/jupyterlite/jupyterlite/pull/1621 is released.